rollup.config.js

```diff
@@ -79,6 +79,18 @@ export default [
         ],
         plugins: [...plugins],
     },
+    {
+        input: 'src/loader-tracing-headers.ts',
```

Comment:
i don't know if the CDN automatically picks these up... would definitely mean we need to edit copy-javascript script in main repo